X/Xyce/build_tarballs.jl

```diff
@@ -0,0 +1,58 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Xyce"
+version = v"7.2.0"
+
+# Collection of sources required to complete build
+sources = [
+            GitSource("https://github.com/Xyce/Xyce.git", "a61faef4bfb2f36f1aa7cc44264bbbb66fbaac11"),
+            DirectorySource("./bundled")
+          ]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+apk add flex-dev
+cd Xyce
+if [[ $target != i686-linux-gnu ]] && [[ $target != x86_64-linux-gnu ]] ; then
+        atomic_patch -p1 ${WORKSPACE}/srcdir/patches/cross.patch
+fi
+./bootstrap
+cd ..
+mkdir buildx
+cd buildx
+/workspace/srcdir/Xyce/./configure --enable-shared --disable-mpi --prefix=${prefix} LDFLAGS="-L${libdir} -lopenblas" CPPFLAGS="-I/$prefix/include -I/usr/include" --host=${target}
+make -j${nprocs}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+   
+platforms = filter(p -> (arch(p) ∈ ("x86_64", "aarch64", "powerpc64le") && (libc(p) == "glibc")),
```

Comment:
This will be platforms = filter(p -> (!Sys.iswindows(p) && !Sys.isapple(p) && !Sys.isfreebsd(p)),
                   supported_platforms())
The rest of Linux `p` builds and are unaffected by this change. So considering the compute time and resources I'll enable them in the final commit.

Reply:
```julia
p -> (!Sys.iswindows(p) && !Sys.isapple(p) && !Sys.isfreebsd(p))
```
basically means `Sys.islinux` or I am missing something?

Reply:
Yes (mostly). `x86_64-linux-libgfortran4-cxx11` weirdly "fails" to find libADMS, libNeuromodels while uses them and builds Xyce. 

I wanted to list out the ones that are definitely not working and will not be added.